development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
+
+* Grab the installer from https://www.qt.io/download-open-source/
+* Run the installer. You ONLY need the qt-creator package so disable the Qt5 and don’t install Qt libs (we will use the qt4 installed by brew)
+* Once the installer is run open qt-creator
+* Make a directory in your QGIS source tree called ``build``
```

Comment:
@timlinux personally I mostly try to keep instructions as short as possible and let a tool (QtCreator) *do the right thing*.
Keeping them short has the advantage that it's easier to follow and there are less possible pitfalls involved. And I think those who do care will create their custom build directory anyway :)